Fixed #35894 -- Reduced the usage of "patch" when referring to a code contribution. (#18780)
django/django: docs/internals/contributing/triaging-tickets.txt

Unfortunately, not all bug reports and feature requests in the ticket tracker
provide all the :doc:`required details<bugs-and-features>`. A number of
tickets have proposed solutions, but those don't necessarily meet all the

> requirements :ref:`adhering to the guidelines for contributing <contribution-style>`.

Reviewer: ```suggestion
requirements :ref:`adhering to the guidelines for contributing <patch-style>`.
```
I'd keep the link the same

Author: Changed (half begrudgingly 😁 )

* **Accepted + No Flags**

  The ticket is valid, but no one has submitted a pull request for it yet.
  Often this means you could safely start writing a fix for it. This is
  generally more true for the case of accepted bugs than accepted features. A
  ticket for a bug that has been accepted means that the issue has been
  verified by at least one triager as a legitimate bug - and should probably be
  fixed if possible. An accepted new feature may only mean that one triager
  thought the feature would be good to have, but this alone does not represent
  a consensus view or imply with any certainty that a pull request will be
  accepted for that feature. Seek more feedback before writing an extensive
  contribution if you are in doubt.

* **Accepted + Has Pull Request**

  The ticket is waiting for people to review the supplied solution. This means
  checking out the pull request and trying it out, verifying that it contains
  tests and docs, checking that all the automated tests on the pull request are

>   passing, and leaving feedback on the ticket.

Reviewer: ```suggestion
  passing, and leaving feedback.
```
As feedback might be better suited on the PR maybe

Author: I feel like I'm repeating myself, but yes, great point 👍🏻

These tickets are uncommon and overall less useful since they don't describe
concrete actionable issues. They are enhancement requests that we might
consider adding someday to the framework if an excellent contribution is
submitted. They are not a high priority.

Other triage attributes
=======================

A number of flags, appearing as checkboxes in Trac, can be set on a ticket:

Has pull request
----------------

This means the ticket has an associated solution. These will be reviewed to
ensure they adhere to the :doc:`documented guidelines

> <writing-code/pull-requests>`.

Reviewer: ```suggestion
<writing-code/submitting-patches>`.
```

A ticket can be resolved in a number of ways:

* fixed

>       Used once a pull request has been merged and the issue is fixed.

Reviewer: ```suggestion
      Used once a contribution has been merged and the issue is fixed.
```
Maybe even "commit" :thinking:

Author: I went for commit, it seemed (marginally) better to me 👍🏻
